### PR TITLE
UI Changes and fixes, create account page, server example for prod

### PR DESCRIPTION
**Pull request by yheismx**
:)

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+//Middlewares
+app.use(bodyParser.json());
+app.use(cors({
+  origin: '*'
+}));
+
+
+const PORT = process.env.PORT || 3500;
+
+const connection = mysql.createConnection({
```
Comment (luigimannoni) [inline server/index.js:20]: Maybe use env variables for the DB connection, @GrenderG should point or setup the right ones

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+});
+
+connection.connect( error => {
+    if(error) throw error;
+    console.log('DB connection established');
+});
+
+//Routes
+app.post('/api/account/createAccount', (req, res) => {
+    console.log(req);
```
Comment (luigimannoni) [inline server/index.js]: Console log

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    const {username, s, v} = req.body;
+    
+    const sql = `INSERT INTO account
+            (
+                username, s, v
+            )
+            VALUES
+            (
+                ?, ?, ?
+            )`;
```
Comment (luigimannoni) [inline server/index.js]: Needs sanitization for incoming requests or something to strip out odd characters

File: src/app.scss [hunk 4]
```diff
@@ -42,3 +42,4 @@ $body-color: #fff;
 @import './sass/navigation';
 @import './sass/video-background';
 @import './sass/image-background';
+@import './sass//create-account';
```
Comment (luigimannoni) [inline src/app.scss]: Double slash

File: src/pages/createAccount.jsx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    // console.log(userData);
+    const {username, password} = userData;
+
+    const data = register(username, password);
+
+    console.log(userCaptcha);
```
Comment (luigimannoni) [inline src/pages/createAccount.jsx]: Console log

File: src/pages/createAccount.jsx [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+   
+    else if(username === '' || password === '' && validateCaptcha()){
+      setalertRequiredInputs(true);
+      setAlertCaptcha(false);
+    }
+    else{
+      
+      setalertRequiredInputs(false);
+      setAlertCaptcha(false);
+      axios.post('http://localhost:3500/api/account/createAccount', {
```
Comment (luigimannoni) [inline src/pages/createAccount.jsx:75]: Needs the URL of the VPS
Cc: @GrenderG

**Review by luigimannoni — COMMENTED** [review]
I've reviewed some files and left comments, can't test this PR until end of September unfortunately :(

**Comment by yheismx** [comment]
I think everything has been fixed, thanks for the comments! :)

**Comment by GrenderG** [comment]
> I think everything has been fixed, thanks for the comments! :)

Thanks a lot! Will wait until @luigimannoni can test it since I'm not a web dev.

**Review by luigimannoni — APPROVED** [review]
Sorry for the wait @yheismx I'm finally back home :) !

Good to merge for now, still need a bit thinking how @GrenderG can deploy the server script on the VPS.
I'll pickup a couple of changes after the merge since I can't directly edit the PR.

**Pull request merged**